app/src/main/java/org/covidwatch/android/presentation/util/Event.kt

```diff
@@ -0,0 +1,27 @@
+package org.covidwatch.android.presentation.util
```

Comment:
not used and should be deleted

Reply:
[Here](https://medium.com/androiddevelopers/livedata-with-snackbar-navigation-and-other-events-the-singleliveevent-case-ac2622673150) is why it needed. But I never used it